client/src/charts/CircleProportionGraph/CircleProportionGraph.js

```diff
@@ -0,0 +1,167 @@
+import { scaleOrdinal } from "d3-scale";
+import _ from "lodash";
+import React, { Fragment } from "react";
+
+import { DisplayTable, Format } from "src/components/index";
+
+import { newIBCategoryColors } from "src/core/color_schemes";
+
+import { TabularLegend } from "src/charts/legends";
+
+import {
+  InteractiveGraph,
+  create_text_maker_component_with_nivo_common,
+  general_default_props,
+  get_formatter,
+} from "src/charts/wrapped_nivo/wrapped_nivo_common";
+
+import text from "./CircleProportionGraph.yaml";
+
+import "./CircleProportionGraph.scss";
+
+const { text_maker, TM } = create_text_maker_component_with_nivo_common(text);
+
+export class CircleProportionGraph extends React.Component {
+  render() {
+    const {
+      is_money,
+      formatter,
+      height,
+      child_value,
+      child_name,
+      parent_value,
+      parent_name,
+      disable_table_view,
+      table_name,
+    } = this.props;
+
+    const color_scale = scaleOrdinal().range(newIBCategoryColors);
+    const value_formatter = get_formatter(is_money, formatter, true, false);
+
+    const Circles = () => {
+      // arbitrary parent values
+      const minDimension = 180;
+      const parent_radius = minDimension / 2;
+      const parent_cx = minDimension / 2;
+      const parent_cy = minDimension / 2;
+
+      // child circle calculations
+      const child_percent = child_value / parent_value;
+
+      // precaution against smaller child_percent values
+      const true_child_radius = parent_radius * Math.sqrt(child_percent);
```

Comment:
Seems like logic from the old component works, since Rc = Rp * Math.sqrt(Ac / Ap) mathematically.